editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -872,16 +773,20 @@ describe('Select Mode Double Clicking With Fragments', () => {
     checkSelectedPaths(renderResult, [desiredPaths[1]])
 
     await doubleClick()
-    checkFocusedPath(renderResult, desiredPaths[1])
-    checkSelectedPaths(renderResult, [desiredPaths[1]])
+    checkFocusedPath(renderResult, null)
+    checkSelectedPaths(renderResult, [desiredPaths[2]])
 
     await doubleClick()
-    checkFocusedPath(renderResult, desiredPaths[1])
+    checkFocusedPath(renderResult, desiredPaths[2])
     checkSelectedPaths(renderResult, [desiredPaths[2]])
 
     await doubleClick()
-    checkFocusedPath(renderResult, desiredPaths[1])
+    checkFocusedPath(renderResult, desiredPaths[2])
     checkSelectedPaths(renderResult, [desiredPaths[3]])
+
+    await doubleClick()
+    checkFocusedPath(renderResult, desiredPaths[2])
+    checkSelectedPaths(renderResult, [desiredPaths[4]])
   })
   // This test fails because there is a generated component there with a root fragment, which case is not supported
```

Comment:
is it possible this test could work now with these changes?

Reply:
No, this is caused by a limitation of `firstAncestorOrItselfWithValidElementPath`, and that is independent of the changes in this PR